mne/preprocessing/interpolate.py

```diff
@@ -65,3 +70,80 @@ def equalize_bads(insts, interp_thresh=1., copy=True):
         inst.info['bads'] = bads_keep
 
     return insts
+
+
+def interpolate_bridged_electrodes(inst, bridged_idx):
+    """Interpolate bridged electrode pairs.
+
+    Because bridged electrodes contain brain signal, it's just that the
+    signal is spatially smeared between the two electrodes, we can
+    make a virtual channel midway between the bridged pairs and use
+    that to aid in interpolation rather than completely discarding the
+    data from the two channels.
+
+    Parameters
+    ----------
+    inst : instance of Epochs, Evoked, or Raw
+        The data object with channels that are to be interpolated.
+    bridged_idx : list of tuple
+        The indices of channels marked as bridged with each bridged
+        pair stored as a tuple.
+
+    Returns
+    -------
+    inst : instance of Epochs, Evoked, or Raw
+        The modified data object.
+    """
+    _validate_type(inst, (BaseRaw, BaseEpochs, Evoked))
+    montage = inst.get_montage()
+    if montage is None:
+        raise RuntimeError('No channel positions found in ``inst``')
+    pos = montage.get_positions()
+    if pos['coord_frame'] != 'head':
+        raise RuntimeError('Montage channel positions must be in ``head``'
+                           'got {}'.format(pos['coord_frame']))
+    ch_pos = pos['ch_pos']
+    # store bads orig to put back at the end
+    bads_orig = inst.info['bads']
+    inst.info['bads'] = list()
+
+    # make virtual channels
+    virtual_chs = dict()
+    bads = set()
+    data = inst.get_data()
+    for idx0, idx1 in bridged_idx:
+        ch0 = inst.ch_names[idx0]
+        ch1 = inst.ch_names[idx1]
+        bads = bads.union([ch0, ch1])
+        # compute midway position in spherical coordinates in "head"
+        # (more accurate than cutting though the scalp by using cartesian)
+        pos0 = _cart_to_sph(ch_pos[ch0])
+        pos1 = _cart_to_sph(ch_pos[ch1])
+        pos_virtual = _sph_to_cart((pos0 + pos1) / 2)
+        virtual_info = create_info(
+            [f'{ch0}-{ch1} virtual'], inst.info['sfreq'], 'eeg')
+        virtual_info['chs'][0]['loc'][:3] = pos_virtual
+        if isinstance(inst, BaseRaw):
+            virtual_ch = RawArray(data[idx0:idx0 + 1], virtual_info,
```

Comment:
Electrodes are bridged but not exactly the same so maybe you use the average of the two electrodes rather than just the first

Reply:
Thanks for the suggestion and that is slightly more accurate but just to note the consequence of being bridged is that they are very nearly identical.